Source/Core/DolphinWX/MemcardManager.cpp

```diff
@@ -430,7 +430,7 @@ bool CMemcardManager::CopyDeleteSwitch(u32 error, int slot)
 		if (slot != -1)
 		{
 			memoryCard[slot]->FixChecksums();
-			if (!memoryCard[slot]->Save()) PanicAlertT(E_SAVEFAILED);
+			if (!memoryCard[slot]->Save()) PanicAlert(E_SAVEFAILED);
```

Comment:
Or, err, you're right, I think... you probably want to manually expand E_SAVEFAILED (and E_UNK, which is used in the same file).